doc/dev/installation.rst

```diff
@@ -93,6 +93,9 @@ The essential steps are:
         cd catalyst
         pip install -r requirements.txt
 
+        # (Only for conda users) Install the latest C++ library
+        conda install -c conda-forge libstdcxx-ng
+
```

Comment:
Since these instructions are meant to be copy-pastable (most likely without reading), I think we want to be careful unconditionally adding it.

Reply:
Yep, I agree. I view the 'minimal building' guide as being 'minimally complete', so that if someone wants to just copy and paste it should just work.

Instead, I would suggest having a 'warning' box _below_. E.g.,

```rest
These steps should give you the full functionality of Catalyst.

.. warning::

    If using Anaconda or Miniconda, please make sure to upgrade ``libstdcxx-ng``:

    .. code-block:: console

        conda install -c conda-forge libstdcxx-ng

    If not, you will receive the following error when doing X:
```

Reply:
(including the error message, if short, will make this searchable for someone who has the error).